doc-content/enterprise-only/optimizer/quarkus-junit-testing-proc.adoc

```diff
@@ -0,0 +1,113 @@
+[id='proc-quarkus-junit-testing_{context}']
```

Comment:
This file isn't included anywhere. Is that intended?

Reply:
Removed

Reply:
Removed